README.md

```diff
@@ -192,7 +192,8 @@ The CLI currently requires two environment variables:
 | PORT                 | Which port to listen on   | number                                        | 443       |
 | HOST                 | Which host to listen on   | ip or hostname                                | empty (all) |
 | HOST_URL             | Hydra's host URL          | url                                           | "https://localhost:4443" |
-| DATABASE_URL         | PostgreSQL Database URL   | `postgres://user:password@host:port/database` | empty     |
+| DATABASE_URL         | Database URL   | e.g: `postgres://user:password@host:port/database` | empty     |
+| DATABASE             | Database Technology   | string e.g: `postgres` or `rethinkdb` | postgres     |
```

Comment:
Yes it could, no standard way of doing it was implemented so i just chose the easier way. I can fix this today. I think that prefixing it would me much more neat.